pkg/operator/controller/status/controller.go

```diff
@@ -92,6 +103,14 @@ func New(mgr manager.Manager, config Config) (controller.Controller, error) {
 	if err := c.Watch(source.Kind[client.Object](operatorCache, &configv1.ClusterOperator{}, handler.EnqueueRequestsFromMapFunc(toDefaultIngressController), predicate.NewPredicateFuncs(isIngressClusterOperator))); err != nil {
 		return nil, err
 	}
+
+	crdPredicate := predicate.NewPredicateFuncs(func(o client.Object) bool {
+		return o.(*apiextensionsv1.CustomResourceDefinition).Spec.Group == gatewayapiv1.GroupName
```

Comment:
Related to an ongoing Slack discussion:

```suggestion
		group := o.(*apiextensionsv1.CustomResourceDefinition).Spec.Group
		return group == gatewayapiv1.GroupName || group == "gateway.networking.x-k8s.io"
```

(getting the group from imports instead, however)

Reply:
Added the check for `gateway.networking.x-k8s.io` group.
Since I was importing the package only to have that GroupName, I switched to hard coded values to avoid that import. For the `x` groupName I'd have to vendor `sigs.k8s.io/gateway-api/apis/v1alpha1`.

Not sure if that's a valid reason, so also happy to do it from imports if need be.